Update docs for upgrade to React Query v3 (#424)
blitz-js/blitzjs.com: app/pages/docs/use-infinite-query.mdx

<!-- prettier-ignore -->
```js
const [

>   queryResult,

Reviewer: ```suggestion
  pagesOfQueryResults,
```

Because this is pages of results, not just the result

    ...extras
  }
] = useInfiniteQuery(queryResolver, getQueryInputArguments, {
  ...options,
  getNextPageParam: (lastPage, allPages) => lastPage.nextPage,
  getPreviousPageParam: (firstPage, allPages) => firstPage.nextPage,
})
```

### Arguments {#arguments}

- `queryResolver:` A [Blitz query resolver](./query-resolvers)
  - **Required**
- `getQueryInputArguments: (fetchNextPageVariable) => queryInputArguments`
  - **Required**
  - A function that accepts the current page options and returns the
    `queryInputArguments`
  - On the first page load, `fetchNextPageVariable` is `undefined`.
  - For subsequent pages, `fetchNextPageVariable` is whatever is returned
    from `getNextPageParam()`
- `options`
  - Optional

### Options {#options}

The options are identical to the options for the
[useQuery hook](./use-query#options) with the addition of the following:

- `getNextPageParam: (lastPage, allPages) => unknown | undefined`
  - When new data is received for this query, this function receives both
    the last page of the infinite list of data and the full array of all
    pages.
  - It should return a **single variable** that will be passed as the last

>     optional parameter to `getQueryInputArguments()`.

Reviewer: This change needs reverted. There's only one argument passed to `getQueryInputArguments`

  - Return `undefined` to indicate there is no next page available.
- `getPreviousPageParam: (firstPage, allPages) => unknown | undefined`
  - When new data is received for this query, this function receives both
    the first page of the infinite list of data and the full array of all
    pages.

>   - It should return a **single variable** that will be passed as the last
>     optional parameter to your `getQueryInputArguments()`.

Reviewer: ```suggestion
  - It should return a **single variable** that will be passed as the
    argument to your `getQueryInputArguments()`.
```

  - Return `undefined` to indicate there is no previous page available.

### Returns {#returns}

> `[queryResult, queryExtras]`

Reviewer: ```suggestion
`[pagesOfQueryResults, queryExtras]`
```

> ##### `queryResult: TData[]`

Reviewer: ```suggestion
##### `pagesOfQueryResults: TData[]`
```
